samples/charts/bar/multi-axis.html

```diff
@@ -75,20 +75,19 @@
 						intersect: true
 					},
 					scales: {
-						yAxes: [{
-							type: 'linear', // only linear but allow scale type registration. This allows extensions to exist solely for log scale for instance
+						y: {
+							type: 'linear',
 							display: true,
 							position: 'left',
-							id: 'y-axis-1',
-						}, {
-							type: 'linear', // only linear but allow scale type registration. This allows extensions to exist solely for log scale for instance
+						},
+						y1: {
```

Comment:
`y` and `y1` is not quite what I'd expect. Would `y1` and `y2` work? (same in the other samples)